layouts/partials/head.html

```diff
@@ -23,11 +25,13 @@
               date.setMonth(date.getMonth() + 1)
             ) {
               var yearMonth = date.toISOString().slice(0, 7);
-              promises.push(
-                fetch("/events/" + yearMonth + ".json")
-                  .then((res) => res.json())
-                  .catch(() => [])
-              );
+              if (eventsMonths.includes(yearMonth)) {
```

Comment:
If we return the `Set`, we can use the `has` method to make this check more efficient. It's not going make a difference now but will do once `N` gets larger: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set#performance

```suggestion
              if (eventsMonths.has(yearMonth)) {
```